aicsimageio/readers/czi_reader.py

```diff
@@ -0,0 +1,740 @@
+                channel_contrast = channel_ce.text if channel_ce is not None else ""
+                scene_channel_list.append(
+                    (f"{channel_id}" f"--{channel_name}" f"--{channel_contrast}")
+                )
+
+            # Attach channel names to coords
+            coords[DimensionNames.Channel] = scene_channel_list
+
+        # Unpack short info scales
+        list_xs = xml.findall(".//Distance[@Id='X']")
+        list_ys = xml.findall(".//Distance[@Id='Y']")
+        list_zs = xml.findall(".//Distance[@Id='Z']")
+
+        scale_xe = list_xs[0].find("./Value")
+        scale_ye = list_ys[0].find("./Value")
+        scale_ze = None if len(list_zs) == 0 else list_zs[0].find("./Value")
+        scale_x = float(str(scale_xe.text)) if scale_xe is not None else 1.0
+        scale_y = float(str(scale_ye.text)) if scale_ye is not None else 1.0
+        scale_z = float(str(scale_ze.text)) if scale_ze is not None else 1.0
+        scale_t = None
+
+        # Handle Spatial Dimensions
+        if scale_ze is not None:
+            coords[DimensionNames.SpatialZ] = np.arange(
+                0,
+                image_short_info[DimensionNames.SpatialZ][1] * scale_z,
+                scale_z,
+            )
+        if scale_y is not None:
+            coords[DimensionNames.SpatialY] = np.arange(
+                0,
+                image_short_info[DimensionNames.SpatialY][1] * scale_y,
+                scale_y,
+            )
+        if scale_x is not None:
+            coords[DimensionNames.SpatialX] = np.arange(
+                0,
+                image_short_info[DimensionNames.SpatialX][1] * scale_x,
+                scale_x,
+            )
+
+        # Time
+        if scale_t is not None:
+            coords[DimensionNames.Time] = np.arange(
+                0,
+                image_short_info["dims"].t * scale_t,
+                scale_t,
+            )
+
+        # Create physical pixel sizes
+        px_sizes = types.PhysicalPixelSizes(scale_z, scale_y, scale_x)
+
+        return coords, px_sizes
+
+    def _read_delayed(self) -> xr.DataArray:
+        """
+        Construct the delayed xarray DataArray object for the image.
+
+        Returns
+        -------
+        image: xr.DataArray
+            The fully constructed and fully delayed image as a DataArray object.
+            Metadata is attached in some cases as coords, dims, and attrs.
+
+        Raises
+        ------
+        exceptions.UnsupportedFileFormatError
+            The file could not be read or is not supported.
+        """
+        with self._fs.open(self._path) as open_resource:
+            czi = CziFile(open_resource)
+
+            dims_shape = CziReader._dims_shape_to_scene_dims_shape(
+                dims_shape=czi.get_dims_shape(),
+                scene_index=self.current_scene_index,
+                consistent=czi.shape_is_consistent,
+            )
+
+            img_dims_list = [letter for letter in self.mapped_dims]
```

Comment:
You can just call `list()` on a string.

`list("hello")` -> `["h", "e", "l", "l", "o"]`